### DISCUSSION TITLE
【组队项目】MoveFlow SDK on Sui

### DISCUSSION THREAD

**Discussion by xiegudong45** [creatorsand/.github#38] [Ideas]
项目名：MoveFlow SDK on Sui

项目介绍：为moveFlow-sui开发typescript SDK

角色需求：有一定的编程基础，乐于学习新知识。

有无 Bounty：无，但有机会加入团队

**Comment by fyang1024** [comment]
有兴趣参与

**Reply by ShirleneLiu** [reply]
@xiegudong45 回头项目 repo 贴一下
